var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -80,6 +81,7 @@ class PyJaxlib(PythonPackage, CudaPackage):
         depends_on("py-numpy@1.22:", when="@0.4.14:")
```

Comment:
There seems to be (line 49) a cudnn upgrade missing (see [here](https://github.com/google/jax/commit/15a1985445ddb5054cce277fc074f1aaad2f14e8#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L101)

I cannot add it as a suggestion, but it's the following

```
depends_on("cudnn@9", when="@0.4.29:")
```

Reply:
Good catch.

Do you want me to add you as a maintainer for py-jaxlib too?

Reply:
Yes, makes sense, please go ahead and add me; jax and jaxlib go in one basically